--- httpd-2.2.32.orig/docs/conf/extra/httpd-ssl.conf.in	2015-05-27 11:59:59.000000000 -0700
+++ httpd-2.2.32/docs/conf/extra/httpd-ssl.conf.in	2017-06-15 14:51:05.957518989 -0700
@@ -56,7 +56,7 @@
 #   ensure these follow appropriate best practices for this deployment.
 #   httpd 2.2.30, 2.4.13 and later force-disable aNULL, eNULL and EXP ciphers,
 #   while OpenSSL disabled these by default in 0.9.8zf/1.0.0r/1.0.1m/1.0.2a.
-SSLCipherSuite HIGH:MEDIUM:!MD5:!RC4
+SSLCipherSuite ALL:!aNULL:!ADH:!eNULL:!LOW:!EXP:RC4+RSA:+HIGH:+MEDIUM
 SSLProxyCipherSuite HIGH:MEDIUM:!MD5:!RC4
 
 #  By the end of 2016, only TLSv1.2 ciphers should remain in use.
@@ -83,7 +83,7 @@
 #   Disable SSLv2 and SSLv3 by default (cf. RFC 7525 3.1.1).  TLSv1 (1.0)
 #   should be disabled as quickly as practical.  By the end of 2016, only
 #   the TLSv1.2 protocol or later should remain in use.
-SSLProtocol all -SSLv2 -SSLv3
+SSLProtocol -ALL +TLSv1.2 +SSLv3 +TLSv1
 SSLProxyProtocol all -SSLv2 -SSLv3
 
 #   Pass Phrase Dialog:
